### ISSUE TITLE
Feature request: zero pad integers in stream_tweets2() filenames

### ISSUE THREAD

**Issue by gadenbuie** [mkearney/rtweet#256]
Currently, `stream_tweets2()` appends an integer to the given (or default filename), but because the integer isn't zero padded the output directory ends up looking like this:

```
stream-1.json
stream-10.json
stream-11.json
...
```

Instead, rwteet could zero pad the integer, ex. `stream-000000001.json`. I have a PR ready to submit if desired that uses a width of 9 as a default.

The advantage of not zero-padding is that you don't know exactly how high the integer will get, but this method at least adds a buffer of 10^9 files before the user would have to deal with it.

**Comment by mkearney** [comment]
I need to go back in and work on `stream_tweets2()` more anyway. But this definitely seems doable!

**Comment by gadenbuie** [comment]
Sounds good! If it helps, I replaced https://github.com/mkearney/rtweet/blob/4f3c44f23b0f273177666663cfb62e0399db5a45/R/stream.R#L424

with

```r
dots[["file_name"]] <- file.path(dir, sprintf("%s-%09.0f.json", file_name, i))
```

I used `%09.0f` instead of `%09d` because big integers in R are represented as numerics. This can probably be tweaked. I don't have enough experience with `stream_tweet()` to know how many files are expected in a long running stream.
